.gitignore

```diff
@@ -7,4 +7,5 @@ prebuilds/
 lib/
 libexec/
 share/
+include/
```

Comment:
> The SemaphoreCI build environment has a bit more space than that. :)
> Let me know if I can help you out CI wise.

Oh, that would be great! :-D Ok, not sure what's the actual space limits, but i was getting a bad run-out-of-space error, that got fixed when I explicitly deleted all the intermediate build files after the toolchain was generated. Ugly hack, but did the job.

@ervinb, is there any place where we can record some suggestions for SemaphoreCI itself? I would like to see a `.travis.yml`-like config file (not only a web interface) and an after-all-parallel-jobs hook so I could speed up a bit the images build, and some other minor things :-)